### ISSUE TITLE
Code error?

### ISSUE THREAD

**Issue by honggyuchoi** [THU17cyz/3DIoUMatch-PVRCNN#7]
Hello, @THU17cyz 

https://github.com/THU17cyz/3DIoUMatch-PVRCNN/blob/1c4c525b7d3642021d6b67bb6fc6da027959a8ac/pcdet/models/detectors/pv_rcnn_ssl.py#L112-L119

I think this part has error.
'ori_boxes[i]' should be revised to 'ori_boxes[inds]' 

A loop started from 116, 
`diff = max_pseudo_box_num - ori_boxes[inds].shape[0] `
`new_box = torch.cat([ori_boxes[inds], torch.zeros((diff, 8), device=ori_boxes[inds].device)], dim=0) `
`new_boxes[inds] = new_box`

Could you check this for me?

**Comment by yezhen17** [comment]
Yes you're right. Since labeled mask is by default [0] in my setting, I didn't noticed this bug. Thank you very much for pointing this out. Feel free to open a PR to fix this.
